definitions/checks/repositories/validate.rb

```diff
@@ -2,6 +2,9 @@ module Checks::Repositories
   class Validate < ForemanMaintain::Check
     metadata do
       description 'Validate availability of repositories'
+      preparation_steps do
+        Procedures::Packages::Install.new(:packages => [ForemanMaintain::Utils::Facter.package])
```

Comment:
Do we actually need this now? There is also a logical issue, as `ForemanMaintain::Utils::Facter.package` actually checks on the presence of the `/usr/bin/facter` which, in case the facter is not installed, would never be true.

We either need to decide which package to install by other ways, assume that some puppet will always be there, or implement the methods we rely on Facter right now with the use of other tools: especially if we need only the os_major_release, there might be simpler ways to achieve that.

Reply:
Good catch!
Yeah, I realized that I made mistake in finding which package to install. But it was too late, I saw your review comment :-)
I will fix that.

In case of os_major_release, facter command itself handles many conditions like different distros. 
For other way, we will need to handle those conditions in our logic. Facter command makes  that easy.

Reply:
Thank you @iNecas.
I have checked puppet version to decide package to install. Updated the PR.